server/services/gameLogic.js

```diff
@@ -185,4 +185,4 @@ const playerHasVoted = (username) => votes.filter(vote => vote.username === user
 const allPlayersPlayedCard = () => players.every(player => playerHasPlayedCard(player.username));
 
 /* Returns true if all players (apart from the current player) have voted this round */
-const allPlayersVoted = () => players.filter(player => player.username !== currentPlayer.username).every(player => playerHasVoted(player));
+const allPlayersVoted = () => players.filter(player => player.username !== currentPlayer.username).every(player => playerHasVoted(player.username));
```

Comment:
```suggestion
const allPlayersVoted = () => players.every(player => player.username === currentPlayer.username || playerHasVoted(player.username));
```
You don't need to loop twice this way. However, it won't have much impact so if you want to leave the `filter` in for clarity, go for it.